.circleci/config.yml

```diff
@@ -91,6 +92,29 @@ jobs:
                         -r requirements/test.pip \
                         -r requirements/develop.pip
             pip install -e .
+      - run:
+          name: Set buildno var
+          command: |
```

Comment:
Maybe add an echo at the end for logging purposes? :)

Reply:
https://app.circleci.com/pipelines/github/etalab/udata-front/2944/workflows/79e83efd-cde7-4ecc-9aa3-f2f9ad8e6ed8/jobs/15120/parallel-runs/0/steps/0-104